vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/resources/META-INF/resources/frontend/menubarConnector.js

```diff
@@ -110,9 +116,9 @@ import './contextMenuConnector.js';
     };
   }
 
-  function setClassName (component) {
-    if (component._item) {
-      component._item.className = component.className;
+  function setClassName(component) {
```

Comment:
Would it be a feasible fix if this was changed into:
```js
function setClassName(component, className) {
  if (component._item) {
    component._item.className = className;
  }
}
```

and called with `...menubarConnector.setClassName(this, $0)", getClassName());`

Reply:
Ah, it wouldn't work in some other cases. Nevermind

Reply:
Yeah, because `component._item` is reassigned to a new item when it becomes part of the overflow.